test/new-e2e/tests/orchestrator/ecs_test.go

```diff
@@ -0,0 +1,158 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package orchestrator
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/pulumi/pulumi/sdk/v3/go/auto"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/DataDog/agent-payload/v5/process"
+	"github.com/DataDog/datadog-agent/test/fakeintake/aggregator"
+	fakeintake "github.com/DataDog/datadog-agent/test/fakeintake/client"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/components"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/runner"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/utils/infra"
+	"github.com/DataDog/test-infra-definitions/scenarios/aws/ecs"
+)
+
+type ecsSuite struct {
+	suite.Suite
+	FakeIntake     *fakeintake.Client
+	ecsClusterName string
+}
+
+func (suite *ecsSuite) GetFakeIntakeClient() *fakeintake.Client {
+	return suite.FakeIntake
+}
+
+func TestECSSuite(t *testing.T) {
+	suite.Run(t, &ecsSuite{})
+}
+
+func extraAgentEnv() string {
+	envs := []string{
+		"DD_ECS_TASK_COLLECTION_ENABLED=true",
+		"DD_ORCHESTRATOR_EXPLORER_ENABLED=true",
```

Comment:
This should be enabled by default. Wonder if we still need to do this

Reply:
https://github.com/DataDog/datadog-agent/pull/25182/commits/37e92ba8792a02e5f7ee6ab5cbe2aad7a24c1b2e